app/services/outbound_health_checker.rb

```diff
@@ -4,10 +4,13 @@ module OutboundHealthChecker
 
   # @return [HealthCheckSummary]
   def check
-    HealthCheckSummary.new(healthy: true, result: outbound_response)
-  rescue StandardError => err
-    NewRelic::Agent.notice_error(err)
-    HealthCheckSummary.new(healthy: false, result: err.message)
+    jitter = rand(-5..5)
+    Rails.cache.fetch('outbound_health_check', expires_in: (45 + jitter).seconds) do
```

Comment:
by default this writes to the filesystem, and in Dashboard app we had some permissions issues

should we make this redis-based so that it is shared across hosts?

https://github.com/18F/identity-dashboard/pull/421

Reply:
Ah, I had assumed the default was memory. Updated in f76c805.

I'm open to Redis, but I think it's safe enough to use memory for now?